snap/backend/src/services/symbolClient.js

```diff
@@ -81,6 +81,50 @@ const symbolClient = {
 				} catch (error) {
 					throw new Error(`Failed to fetch mosaics namespace: ${error.message}`);
 				}
+			},
+			/**
+			 * Fetch transactions page by address with offset id.
+			 * @param {string} address - The address to account.
+			 * @param {string} offsetId - The offset id of transaction.
+			 * @param {'confirmed' | 'unconfirmed'} group - The group of transactions.
```

Comment:
Will partial group be supported also?

Reply:
Nope, right now only for `confirmed`.

For partial group I think, it should implement together with multisign account